packages/styles/src/components/blockquote.scss

```diff
@@ -0,0 +1,34 @@
+@use '../tokens/components';
+@use '../functions/tokens';
+@use './../placeholders/blockquote' as blockquote-ph;
+
+tokens.$default-map: components.$post-quote;
+
+.blockquote {
```

Comment:
I am just wondering if the blockquote is a component or an element since the `.blockquote` class should only be used on `<blockquote>` elements.... 
I am adding this as a topic for the devs roundtable.